cmd/server.go

```diff
@@ -84,5 +89,49 @@ func serverRun(conf *config.Config) {
 		c.FileFromFS("/", assets.EmbedFS())
 	})
 
-	ginutil.Startup(ge, ":8222")
+	if conf.TLS.Enabled {
+		tlsAddr := fmt.Sprintf(":%d", conf.Server.SSLPort)
+		go startTls(ge, tlsAddr, conf.TLS.Auto, conf.TLS.CacheDir, conf.Server.Domain, conf.TLS.CertPath, conf.TLS.CertkeyPath)
+	}
+
+	addr := fmt.Sprintf(":%d", conf.Server.Port)
+	ginutil.Startup(ge, addr)
+}
+
+func startTls(e *gin.Engine, addr string, auto bool, cacheDir string, domain []string, cert, key string) {
```

Comment:
It is best not to define so many parameters, you can directly pass in conf

Reply:
changed the code